### ISSUE TITLE
Opening image as toot is deleted crashes masto frontend

### ISSUE THREAD

**Issue by clarjon1** [tootsuite/mastodon#15860]
### Expected behaviour

Masto frontend to not crash. Perhaps show a message like "The image has been removed" or something instead?

### Actual behaviour

Open an image as an incoming delete on a toot
![image](https://user-images.githubusercontent.com/627845/110374496-bccf2b80-801e-11eb-9fb2-dddedc6a37e4.png)
this appears

STacktrace:

```TypeError: r is null


a</t.prototype.render@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:256798
Bi@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:1016651
qi@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:1016446
Os@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:1052060
bu@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:1043503
mu@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:1043426
su@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:1040456
Ya/<@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:992097
t.unstable_runWithPriority@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:1067155
Va@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:991806
Ya@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:992044
Xa@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:991977
uu@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:1040747
notify@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:86223
c</t.notifyNestedSubs@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:86858
c</t.handleChangeWrapper@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:86926
g@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:404407
Ot/</</<@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:376149
Ot/</</<@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:375693
Ot/</</</<@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:375446
a/</</<@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:330633
dispatch@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:406578
w/<@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:133508
a/</</<@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:330624
dispatch@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:406578
onReceive@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:661432
received/<@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:659155
received@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:659132
m/l.onmessage@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:660361
value@https://redroo.ml/packs/js/common-a34d4fc19bd4731b9983.js:2:555816



render (webpack:///app/javascript/mastodon/features/picture_in_picture/components/footer.js:122:45)
zh (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:3962:72)
zh (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:3952:4)
si (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:6184:6)
b.forEach(function (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5549:6)
Ui (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5545:18)
c.parentNode.insertBefore (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5352:400)
dc (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:2254:4)
e (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:7162:10)
tc (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:2237:55)
dc (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:2251:8)
dc (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:2248:73)
Mi (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5367:4)
batch(function (webpack:///node_modules/react-redux/es/utils/Subscription.js:19:6)
notifyNestedSubs (webpack:///node_modules/react-redux/es/utils/Subscription.js:90:19)
handleChangeWrapper (webpack:///node_modules/react-redux/es/utils/Subscription.js:95:11)
Error (webpack:///node_modules/redux/es/redux.js:213:36)
audio.pause (webpack:///app/javascript/mastodon/middleware/sounds.js:15:21)
action.type.match (webpack:///app/javascript/mastodon/middleware/errors.js:10:17)
action.type.match (webpack:///app/javascript/mastodon/middleware/loading_bar.js:18:29)
function (webpack:///node_modules/redux-thunk/es/index.js:5:11)
_dispatch (webpack:///node_modules/redux/es/redux.js:634:24)
deleteFromTimelines (webpack:///app/javascript/mastodon/actions/timelines.js:53:13)
function (webpack:///node_modules/redux-thunk/es/index.js:5:4)
_dispatch (webpack:///node_modules/redux/es/redux.js:634:24)
onReceive (webpack:///app/javascript/mastodon/actions/streaming.js:76:19)
hashtag:local'].includes (webpack:///app/javascript/mastodon/stream.js:103:67)
hashtag:local'].includes (webpack:///app/javascript/mastodon/stream.js:102:39)
streamingAPIBaseURL.startsWith (webpack:///app/javascript/mastodon/stream.js:233:86)
value (webpack:///node_modules/@gamestdio/websocket/lib/index.js:116:11)
```

### Specifications

3.3.0

Firefox 86.0, win10

**Comment by clarjon1** [comment]
Note: This has happened a few times, this is just the time it annoyed me enough to report.

**Comment by ClearlyClaire** [comment]
Fixed by #15701
